pages/index.tsx

```diff
@@ -1,11 +1,41 @@
 import React from 'react';
 import Layout from '../components/Layout';
+import LobbyCircle  from '../components/LobbyCircle';
+import styles from '../styles/home.module.scss';
+
+const SAMPLE_LOBBIES = [
+  'jeff',
+  'annie',
+  'troy',
+  'abed',
+  'britta',
+  'shirley',
+  'pierce',
+  'chang',
+  'craig',
+  'starburns',
+  'magnitude',
+  'ian',
+  'buzz',
+];
 
 export default function Home(): JSX.Element {
+  let empty = true;
+  if (SAMPLE_LOBBIES.length > 0) {
+    empty = false;
+  }
+
+  const createButton = empty ? <LobbyCircle name='CREATE' create={1}/>
+    : <LobbyCircle name='CREATE' create={2}/>;
```

Comment:
Get rid of the logic on L23-L26

```suggestion
  const createButton = <LobbyCircle name='CREATE' create={!SAMPLE_LOBBIES.length ? 1 : 2}/>
```